core/src/components/radio/radio.common.scss

```diff
@@ -156,6 +156,17 @@ input {
   display: block;
 }
 
+// Radio Label
+// ----------------------------------------------------------------
+
+.label-text-wrapper {
+  text-overflow: ellipsis;
+
+  white-space: nowrap;
+
+  overflow: hidden;
+}
```

Comment:
Noticed that the select options weren't truncating with ellipsis. Moved this to common since every theme behaves the same.